examples/sample-app/README.md

```diff
@@ -3,7 +3,11 @@ OpenShift 3 Application Lifecycle Sample
 
 This is a set of configuration files and scripts which work with OpenShift 3 to create a new application and perform application builds.
 
-This example assumes you have successfully built the `openshift` binary executable and have Docker installed/working.  See https://github.com/openshift/origin/blob/master/CONTRIBUTING.adoc.
+This example assumes you have successfully built the `openshift`
+binary executable, and have Docker installed/working.  See
+https://github.com/openshift/origin/blob/master/CONTRIBUTING.adoc. `osc`
+should be either a symlink or a copy of `openshift` (it serves as a
+shortcut for `openshift cli`).
```

Comment:
It doesn't necessarily put osc in the user's path (or root's), and given this is likely to be read by someone who's pretty new, I'd rather not assume they know the relationship between osc and openshift. Though I agree the setup instructions really ought to go through all this clearly enough... CONTRIBUTING.md and README.md need some work too (nothing called "contributing" should be required reading for setting up standard examples).